databases/week1/homeworkDTB.txt

```diff
@@ -0,0 +1,55 @@
+-- 1. all number of tasks, 35
+SELECT COUNT(id)
+FROM homework1.task;
+
+-- 2. number of tasks whose due date is null, 8
+SELECT COUNT(*)
+FROM homework1.task
+WHERE due_date IS null;
+
+-- 3. all below tasks are done
+SELECT *
+FROM homework1.task
+WHERE status_id = 3;
+
+-- 4. all the tasks that are not done
+SELECT *
+FROM homework1.task
+WHERE status_id != 3;
+
+-- 5. All tasks sorted from recent most
+SELECT *
+FROM homework1.task 
+ORDER BY created DESC;
+
+-- 6. single most created task
+SELECT *
+FROM homework1.task 
+ORDER BY created DESC
+LIMIT 1;
+
+-- 7. tasks titles and their due dates whose contains the name database
+SELECT title, due_date
+FROM homework1.task
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+
+-- 8. getting the title and status of all tasks.
+SELECT task.title, status.name 
+FROM homework1.task 
+INNER JOIN status ON task.status_id = status.id;
+
+-- 9. Get the name of each status and the count of how many tasks with status. 
+SELECT *
+FROM task
+INNER JOIN status ON task.status_id = status.id;
+
+-- 10. get the name of all status, sorted by status with the most task first
+SELECT COUNT(id)
+FROM task
+JOIN status.id 
+ON task.status.id = status.status.id;
```

Comment:
All the previous queries are correct. You only have mistakes in these two. Remember you need to list the three statuses along with the number of tasks having that status. Also as a hint I can say that the two last queries should look almost identical, except for the last part. Let me know if you need help